nautobot/ipam/tests/test_models.py

```diff
@@ -1352,6 +1351,10 @@ def test_assign_vlan_out_of_range(self):
             vlan.validated_save()
         self.assertEqual(str(exc.exception), "{'vid': ['VLAN ID is not contained in VLAN Group range (1-2)']}")
 
+        # Next out of range.
+        VLAN.objects.bulk_create((VLAN(name="VLAN 6", vid=6, vlan_group=vlangroup, status=status),))
+        self.assertEqual(vlangroup.get_next_available_vid(), None)
+
```

Comment:
Is this correct or is it a merge artifact duplicate of lines 1319-1321?

Reply:
It is a merge duplicate thanks for catching it!